pootle/apps/pootle_store/models.py

```diff
@@ -1196,7 +1197,7 @@ def addunit(self, unit, index=None, user=None, update_revision=None,
         if self.id:
             newunit.save(
                 revision=update_revision,
-                user=user,
+                created_by=user,
```

Comment:
afaict - i put a pdb in the save method and ran the full test suite to catch any